### PR TITLE
makefile: .odb, .log and .json now share stem

### PR DESCRIPTION
**Pull request by oharboe**

### CODE REVIEW

File: flow/Makefile [hunk 1]
```diff
@@ -475,18 +475,18 @@ endif
 # STEP 4: Macro Placement
 #-------------------------------------------------------------------------------
 $(RESULTS_DIR)/2_4_floorplan_macro.odb: $(RESULTS_DIR)/2_3_floorplan_tdms.odb $(RESULTS_DIR)/1_synth.v $(RESULTS_DIR)/1_synth.sdc $(MACRO_PLACEMENT) $(MACRO_PLACEMENT_TCL)
-	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/macro_place.tcl -metrics $(LOG_DIR)/2_4_mplace.json) 2>&1 | tee $(LOG_DIR)/2_4_mplace.log
+	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/macro_place.tcl -metrics $(LOG_DIR)/floorplan_macro.json) 2>&1 | tee $(LOG_DIR)/floorplan_macro.log
```
Comment (maliberty) [inline flow/Makefile]: Lost the numeric prefix

File: flow/Makefile [hunk 2]
```diff
@@ -780,7 +780,7 @@ else
 endif
 
 $(RESULTS_DIR)/6_final.cdl: $(RESULTS_DIR)/6_final.v
-	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/cdl.tcl) 2>&1 | tee $(LOG_DIR)/6_cdl.log
+	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/cdl.tcl) 2>&1 | tee $(LOG_DIR)/6_final.log
```
Comment (maliberty) [inline flow/Makefile]: Cdl is not the final report step

**Comment by oharboe** [comment]
@maliberty Some unrelated failures, will inspect rest when it is finished...

```
Loaded plugins: fastestmirror, ovl
Cannot open: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm. Skipping.
Error: Nothing to do
```

File: flow/Makefile (hunk 1 above)
Comment (rovinski) [inline flow/Makefile]: Why remove numbering?

File: flow/Makefile [hunk 3]
```diff
@@ -621,7 +621,7 @@ $(RESULTS_DIR)/5_2_route.odb: $(RESULTS_DIR)/5_1_grt.odb
 else
 $(RESULTS_DIR)/5_2_route.odb: $(RESULTS_DIR)/4_cts.odb
 endif
-	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/detail_route.tcl -metrics $(LOG_DIR)/5_2_TritonRoute.json) 2>&1 | tee $(LOG_DIR)/5_2_TritonRoute.log
+	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/detail_route.tcl -metrics $(LOG_DIR)/5_2_route.json) 2>&1 | tee $(LOG_DIR)/5_2_route.log
```
Comment (rovinski) [inline flow/Makefile:624]: Why grt and route instead of grt and drt, or route_global and route_detailed?

File: flow/Makefile (hunk 2 above)
Comment (rovinski) [inline flow/Makefile]: did you mean 6_final_cdl?

**Review by maliberty — APPROVED** [review]
The step naming is overall inconsistent but at least the common stem is fixed.

**Comment by maliberty** [comment]
The CI has real errors:
```
Error: field detailedplace__design__violations, has invalid float format for build_value
```

**Comment by oharboe** [comment]
> The CI has real errors:
> 
> ```
> Error: field detailedplace__design__violations, has invalid float format for build_value
> ```

@maliberty I missed a spot. Fixed.

**Pull request merged**